ISISNeutronMuon/MDANSE
# Lammps converter fix

**Description of work**
The existing LAMMPS converter works correctly only for a small number of specific cases. LAMMPS gives the user multiple options of the physical unit system output file format.

The converter has been extended to allow the user to pick the unit system, and supports three trajectory formats: LAMMPS custom format (this has been considered to be the ".lammps" format so far), XYZ and LAMMPS implementation of H5MD.

In principle, another change will be necessary in the future to account for different `atom_style` options, which will change the format of the structure file (known in MDANSE as the 'config' file).

closes #421 

**Fixes**
Added multiple readers to the LAMMPS converter.
Added the unit system selector.
Included additional trajectories for unit testing, together with the input files used to generate them.
Added new unit tests for the LAMMPS converter.

**To test**
All tests must pass.
Also, please try to convert a LAMMPS trajectory using the protos branch and again using this branch. The results should be the same.
Optionally, please try to manually convert the new trajectory files from unit test data and check that they all result in the same output.
